### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute error plus rar unavailable in v1.6.0a4

### ISSUE THREAD

**Issue by EweSparky** [comictagger/comictagger#459]
Hi there,

When I try to run `comictagger --copy CBL --type CR ./*.cbz` in v1.6.0a4 I get the following error:

```
2023-04-28T19:42:28 | comictagger | ERROR | CLI mode failed
Traceback (most recent call last):
  File "/home/stuart/.local/lib/python3.9/site-packages/comictaggerlib/main.py", line 218, in main
    cli.CLI(self.config[0], talkers).run()
  File "/home/stuart/.local/lib/python3.9/site-packages/comictaggerlib/cli.py", line 182, in run
    self.process_file_cli(f, match_results)
  File "/home/stuart/.local/lib/python3.9/site-packages/comictaggerlib/cli.py", line 587, in process_file_cli
    self.copy(ca)
  File "/home/stuart/.local/lib/python3.9/site-packages/comictaggerlib/cli.py", line 319, in copy
    if self.config.apply_transform_on_bulk_operation_ndetadata_style == MetaDataStyle.CBI:
AttributeError: 'Namespace' object has no attribute 'apply_transform_on_bulk_operation_ndetadata_style'
```

Also, when I run comictagger, I get a `rar unavailable` in the standard output even though `which rar` returns `/usr/bin/rar`. If I try to set the path in the GUI settings, the change is unset as soon as I reopen the settings window. The wiki includes the setting to include in the older `settings.ini` file but not `settings.json`.

**Comment by lordwelch** [comment]
I think I forgot to mention it in the changelog, but the rar dependency has changed. Please install using `pip install "comictagger==1.6.0a4[CBR]"`

**Comment by EweSparky** [comment]
I get the following error when I try that:

```
ERROR: Extras after version '==1.6.0a4[CBR]'.
```

**Comment by lordwelch** [comment]
Sorry wrong syntax `pip install "comictagger[CBR]==1.6.0a4"`

**Comment by EweSparky** [comment]
Success! Thank you. :) And I see `settings.json` has the updated rar setting:

```
  "archiver": {
    "rar": "/usr/bin/rar"
  },
```

**Comment by lordwelch** [comment]
The `AttributeError: 'Namespace' object has no attribute error` was fixed in 783e10a9a180a6dd219c738b9845302c15dfce4c
